modules/activemq-artemis-launch/added/drain.sh

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env bash
 
-export BROKER_HOST="$(hostname -f)"
+if [ -z "${DRAINER_IP}" ]; then
```

Comment:
The name `DRAINER_HOST` would work with hostnames and IP addresses. In the case we decide to pass an hostname form the oeprator.

Reply:
+1

Reply:
I'll change it to DRAINER_HOST